packages/babel-plugin-transform-es2015-function-name/test/fixtures/function-name/await/expected.js

```diff
@@ -0,0 +1,3 @@
+export {};
+
+var obj = { await: function _await() {} };
```

Comment:
In chrome this is happens - so defining await inside async function is syntax error. Not sure if it's a chrome bug ... 

<img width="344" alt="screenshot 2016-12-08 16 32 03" src="https://cloud.githubusercontent.com/assets/294474/21015924/05bc564c-bd64-11e6-8f9d-e5a2762e06f3.png">